diff --git a/build.rs b/build.rs
--- a/build.rs
+++ b/build.rs
@@ -117,7 +117,11 @@ mod build_bundled {
         {
             super::copy_bindings(lib_name, "bindgen_bundled_version", out_path);
         }
-        println!("cargo:include={}/{lib_name}", env!("CARGO_MANIFEST_DIR"));
+        println!(
+            "cargo:include={}/{lib_name}",
+            env::var("CARGO_MANIFEST_DIR")
+                .expect("Cargo must set the build script's current manifest directory")
+        );
         println!("cargo:rerun-if-changed={lib_name}/sqlite3.c");
         println!("cargo:rerun-if-changed=sqlite3/wasm32-wasi-vfs.c");
         let mut cfg = cc::Build::new();
